tests/test_input.py

```diff
@@ -184,10 +184,23 @@ def disc_zero_angle():
 @pytest.fixture
 def disc_xrotation():
     m, pos, vel = solid_disk(N_part=1000)
-    a = rot_matrix_xaxis(theta=0.5 * np.pi * np.random.random())
+    a = rot_matrix_xaxis(theta=0.3 * np.pi * np.random.random())

```

Comment:
here use `random` instead of `np.random` so it actually uses the seed

Reply:
Ok, I will change that